br/pkg/restore/data.go

```diff
@@ -292,50 +300,82 @@ func (recovery *Recovery) WaitApply(ctx context.Context) (err error) {
 	return eg.Wait()
 }
 
-// ResolveData a worker pool to all tikv for execute delete all data whose has ts > resolvedTs
-func (recovery *Recovery) ResolveData(ctx context.Context, resolvedTs uint64) (err error) {
+// get the region key range
+func (recovery *Recovery) getRegionKeyRanges() []tikvstore.KeyRange {
```

Comment:
can we use a large key range rather than region key ranges to reduce the gRPC invoke times?

Reply:
The purpose is to get all-regions keyRange in order to have a parallel flashback as the as tidb ddl done.

Reply:
fix in latest commit